npm/src/upload/form/upload-form.ts

```diff
@@ -131,6 +131,20 @@ export class UploadForm {
     displaySelectionSuccessMessage(this.successMessage, this.warningMessages)
   }
 
+  removeSelectedItem = (): void => {
```

Comment:
We talked about this on Slack but you need to reset the form here and add an event in to prevent the url changing. Something like 
```
removeSelectedItem = (ev: Event): void => {
  ev.preventDefault()
  const successMessageRow: HTMLElement | null = document.querySelector(
    "#success-message-row"
  )
  this.selectedFiles = []
  successMessageRow?.setAttribute("hidden", "true")
  this.formElement.reset()
}
```